.github/workflows/test_package.yml

```diff
@@ -22,69 +22,57 @@ jobs:
     # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
     runs-on: ubuntu-22.04
 
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, "3.10"]
```

Comment:
```suggestion
        python-version: ["3.8", "3.9", "3.10"]
```
for consistency

Reply:
Fixed in https://github.com/jorgensd/reproducibility/pull/33/commits/660ce5168d4343250337bbf24d42fdfb554e696a